addons/pos_hr/static/tests/tours/utils/pos_hr_helpers.js

```diff
@@ -54,3 +54,9 @@ export function login(name, pin) {
         Dialog.confirm(),
     ]);
 }
+export function clickLockButton() {
+    return {
+        content: "Click on the menu button",
+        trigger: ".pos-rightheader button.fa-lock",
```

Comment:
```suggestion
        trigger: ".pos-rightheader button.fa-lock",
        run: "click",
```
there is a new change in the tour runner. "No action" is now the default of each tour step.